src/main/java/org/folio/circulation/domain/policy/OverdueFinePolicy.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.circulation.domain.policy;
+
+import io.vertx.core.json.JsonObject;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+public class OverdueFinePolicy extends CirculationPolicy {
+
```

Comment:
Is this blank line needed? Does it help with readability?

Reply:
It seems to me it is more readable. The blank line divided the definition of the class and the constructor

Reply:
Ok, I typically don't find that. I'll resolve this.